chat/tests.py

```diff
@@ -1,3 +1,102 @@
 from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+from .models import ChatBot, Conversation, Message
+from constants.header_constants import HEADER_USER_EMAIL
 
-# Create your tests here.
+
```

Comment:
It seems like there are two new lines here, if so please reduce to one

Reply:
That's a feature not a bug! :P  https://peps.python.org/pep-0008/#blank-lines

Reply:
It is indeed! I didn't know this, thanks for sharing 😄